structs/structs.c

```diff
@@ -8,7 +8,10 @@
     Person should have the fields `name`, `age`, `height`, and `weight`.
 */
 typedef struct Person {
-
+    char name[20];
+    int age;
```

Comment:
Loving your progress Simon keep it up!, and let me know if you have any questions